src/scalars/aliases.ts

```diff
@@ -1,5 +1,5 @@
 import { GraphQLFloat, GraphQLString, GraphQLInt, GraphQLScalarType } from "graphql";
 
-export const Int = GraphQLInt;
-export const Float = GraphQLFloat;
-export const ID = GraphQLString;
+export const Int: GraphQLScalarType = GraphQLInt;
```

Comment:
I don't like explicit types, I always use `noImplicitAny` and rely on type inference. Especially in this case when it's too verbose and redundant.

Reply:
it's for the readability and introduction easier phase in for new contributors